feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.service.ts

```diff
@@ -23,8 +23,8 @@ export class ConfiguratorAttributeQuantityService {
    * @returns Render a quantity component?
    */
   withQuantity(
-    dataType: Configurator.DataType,
-    uiType: Configurator.UiType
+    dataType: Configurator.DataType | undefined,
+    uiType: Configurator.UiType | undefined
```

Comment:
Could you add a unit test where you call this method with undefined parameters and check wether or not the method behaves as intended.

Reply:
Not needed, as I could remove `| undefined` here